### PR TITLE
Add upgrade testnet info for May 15, 2020 upgrade.

### PR DESCRIPTION
**Pull request by Mengerian**
Also amend related links and and info.

### CODE REVIEW

**Comment by Mengerian** [comment]
@majcosta Please have a look also

**Comment by majcosta** [comment]
Looks good to me.

**Comment by jplg1986** [comment]
Good day BitcoinCash
I am busy working on my own project but i need a little bit of funding 
Any help will be appreciated 

Wallet for funding - qp4rrq7j7rsmc87f43tpy63xrf9yrkxpxsusm6k9e9

0x512630DC263FD4c71DBe81Fec68cF61156d79E80

File: workgroups/wg-testing/2020-05-15_upgrade_testnet.md [hunk 1]
```diff
@@ -7,21 +7,20 @@ To assist technical preparations for the upgrade, a testnet is available where t
 
 ## Upgrade Testnet Parameters
 
-Fork time: `1582729200`  (February 26, 2020 15:00:00 UTC)
+Fork time: `1586966400`  (April 15, 2020 16:00:00 UTC)
 
 ## Set up a node
 
-Run Bitcoin ABC 0.21.0 with:
+Run Bitcoin ABC 0.21.3 with:
```
Comment (majcosta) [inline workgroups/wg-testing/2020-05-15_upgrade_testnet.md:14]: Should we make that 0.21.4 given that has already been released?

File: workgroups/wg-testing/2020-05-15_upgrade_testnet.md [hunk 2]
```diff
@@ -7,21 +7,20 @@ To assist technical preparations for the upgrade, a testnet is available where t
 
 ## Upgrade Testnet Parameters
 
-Fork time: `1583427600`  (March 05, 2020 17:00:00 UTC)
+Fork time: `1586966400`  (April 15, 2020 16:00:00 UTC)
 
 ## Set up a node
 
-Run Bitcoin ABC 0.21.1 with:
+Run Bitcoin ABC 0.21.3 with:
```
Comment (majcosta) [inline workgroups/wg-testing/2020-05-15_upgrade_testnet.md:14]: Maybe we should bump this up to 0.21.4 since it's released already?

Comment (Mengerian) [reply workgroups/wg-testing/2020-05-15_upgrade_testnet.md]: OK, done

**Pull request merged**